employees/models.py

```diff
@@ -15,8 +15,17 @@
 from users.models import CustomUser
 
 
+class TaskActivityTypeQuerySet(models.QuerySet):
+    def filter_default_task_activities(self) -> QuerySet:
```

Comment:
I think `get_defaults` should be enough.

Reply:
Done